cli/script/command-executor.ts

```diff
@@ -1288,6 +1295,19 @@ function throwForInvalidOutputFormat(format: string): void {
 function whoami(command: cli.ICommand): Promise<void> {
     return sdk.getAccountInfo()
         .then((account): void => {
-            log(`${account.email} (${account.linkedProviders.join(", ")})`);
+            log(`Account: ${account.email} (${account.linkedProviders.join(", ")})`);
+
+            var connectionInfo = deserializeConnectionInfo();
+            if (connectionInfo.noProxy) {
+                log(`Proxy: Ignored`);
+            } else if (connectionInfo.proxy) {
+                log(`Proxy: ${connectionInfo.proxy}`);
```

Comment:
Could we use chalk.green to colorize the labels? That way it's consistent with the other key/value pairs we display throughout the CLI.